src/config/rds_aurora.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "Aurora RDS",
+  "displayName": "Amazon Aurora RDS",
+  "rules": [
+    {
+      "aws": {
+        "logGroup": "/aws/rds/cluster/%{DATA:resource_id}/%{GREEDYDATA:log_type}"
+      },
+      "sources": [
+        {
+          "sourceType": "logs",
+          "source": "log_group",
+          "condition": "$prefix('/aws/rds/cluster/')"
+        }
+      ],
+      "attributes": [
+        {
+          "key": "aws.service",
+          "pattern": "'rds'"
+        },
+        {
+          "key": "aws.resource.id",
+          "pattern": "resource_id"
+        },
+        {
+          "key": "log_stream",
+          "pattern": "if(log_type == 'postgresql', &replace_regex(log_stream, '\\.0$', ''), &log_stream, @ )",
+          "priority": 9
+        },
+        {
+          "key": "aws.arn",
+          "pattern": "format_arn('arn:{}:rds:{}:{}:db:{}', [partition, region, account_id, \"log_stream\"])",
+          "priority": 10
+        },
+        {
+          "key": "dt.source_entity",
+          "pattern": "dt_meid_rds_v2(\"aws.arn\")"
+        },
+        {
+          "key": "severity",
+          "pattern": "if(log_type == 'slowquery', &'WARNING', &null, @)",
+          "priority": 11
+        },
+        {
+          "key": "severity",
+          "pattern": "if(log_type == 'postgresql', &if(contains(log_content, ':WARNING:'), &'WARNING', &if(contains(log_content, ':ERROR:'), &'ERROR', &'INFO', @), @), &if(contains(log_content, '[Warning]'), &'WARNING', &if(contains(log_content, '[Error]'), &'ERROR', &null, @), @), @)",
```

Comment:
for all 10 types of pytest cases I tun it's multiply by 1000 so 10 000 requests and got:
```
24930185 function calls (23957927 primitive calls) in 27.175 seconds
```

Reply:
I am not sure I understand. How much data has been transformed in 27seconds? 1MB? 5MB?

Reply:
Is this only for this specific jmespath expression or whole config? Please add more context

Reply:
10 000 test cases as we have right now for all databases processed by pytest

Reply:
from this line https://github.com/dynatrace-oss/dynatrace-aws-log-forwarder/blob/7320460fad726b06abfa15fdc96c5a0b101722f1/tests/unit/logs/test_transformation_full.py#L668 to the bottom multiply 1000 times

Reply:
but I don't have trust to that perf library right now without that additional section it took 26s but with that severity parsing it took 31s

Reply:
This numbers tell me nothing. Perf for this should say following:
I pushed X MB of log entries (number of entries is Y) through this specific json config and it took Z seconds.
I don't care how fast all of our testcases are

Reply:
@mmajcher ok, so where we have performance test that I can check how the performance was degraded or not?

Reply:
The performance surely is degraded but question is how much
Unfortunately we don't have one so if you with @mswiatkowska decide that you feel safe then push as is :)

Reply:
merged your branch and move to separate stage